debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -647,6 +646,14 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                       .withValidation(MySqlConnectorConfig::validateColumnBlacklist)
                                                       .withDescription("");
 
+    public static final Field TABLE_SNAPSHOT_ORDER_SPECIFIER = Field.create("table.snapshot.order.specifier")
+                                                                .withDisplayName("Snapshot table data in the specified order")
+                                                                .withType(Type.STRING)
+                                                                .withWidth(Width.LONG)
+                                                                .withImportance(Importance.LOW)
+                                                                .withValidation(MySqlConnectorConfig::validateTableSnapshotOrderSpecifier)
+                                                                .withDescription("");
```

Comment:
The description should be something you might find in a tool tip or in generated documentation: what does this configuration property do?